projects/helper/coreAssets.json

```diff
@@ -1535,7 +1535,8 @@
     "WETH": "0x4200000000000000000000000000000000000001",
     "USDC": "0x49A5010110a358d9069282873F3e7eCf6B41DC10",
     "USDT": "0x0Cf7c2A584988871b654Bd79f96899e4cd6C41C0",
-    "WEMIX": "0x3720b1dc2c8dde3bd6cfcf0b593d0a2bbcac856e"
+    "WEMIX": "0x3720b1dc2c8dde3bd6cfcf0b593d0a2bbcac856e",
+    "M-BTC": "0x0F921c39eFd98809FE6D20a88A4357454578987a"
```

Comment:
please revert this change

Reply:
I pushed revert commit [#ab5002a](https://github.com/DefiLlama/DefiLlama-Adapters/pull/10947/commits/ab5002a419a0ba9ce92446d91f2c35cd408cf895)